files/en-us/web/html/quirks_mode_and_standards_mode/index.html

```diff
@@ -46,7 +46,7 @@ <h3 id="XHTML">XHTML</h3>
 
 <h2 id="How_do_I_see_which_mode_is_used">How do I see which mode is used?</h2>
 
-<p>In Firefox, select <em>View Page Info</em> from the context menu, and look for <em>Render Mode</em>.</p>
+<p>In Firefox, select <em>Page Info</em> from the <em>Tools</em> context menu, and look for <em>Render Mode</em>.</p>
```

Comment:
```suggestion
<p>In Firefox, select <em>Page Info</em> from the <em>Tools</em> menu bar, and look for <em>Render Mode</em>.</p>
```

Can we agree on the middle ground?
Hitting <kbd>Alt</kbd> brings up the menu bar for me (on Linux - resp. Command on macOS?).

Reply:
Oh! I got confused about what "context menu" was when I didn't see "View Page Info" there: 

<img width="268" alt="Screen Shot 2021-07-27 at 4 37 14 PM" src="https://user-images.githubusercontent.com/3607279/127224068-2be349e7-7452-42d7-bd79-73108d297cee.png"> (Firefox on macOS)

Since "Page Info" is in the menu bar in both Linux and macOS, I think your suggested change is a great way to go. on macOS, the menu bar is shown without having to bring it up.

(By the way, this is the resource I used to find the Page Info window when these instructions didn't work for me: https://support.mozilla.org/en-US/kb/firefox-page-info-window)

Reply:
Ah, you've found it!
I actually hopped over to support.mozilla.org in hope of finding a link to send you. We should include it!
May I ask you for one more round of adjustment to sneak that link in?